docs/src/transaction_fees.md

```diff
@@ -1,21 +1,70 @@
 ---
 title: Transaction Fees
+description: "Transaction fees are the small fees paid to process instructions on the network. These fees are based on computation and an optional prioritization fee."
+keywords: "instruction fee, processing fee, storage fee, low fee blockchain, gas, gwei, cheap network, affordable blockchain"
 ---
 
-**Subject to change.**
+The small fees paid to process [instructions](./terminology.md#instruction) on the Solana blockchain are known as "_transaction fees_".
 
-Each transaction sent through the network, to be processed by the current leader validation-client and confirmed as a global state transaction, contains a transaction fee. Transaction fees offer many benefits in the Solana economic design, for example they:
+As each transaction (aka collection of instructions) is sent through the network, it gets processed by the current leader validation-client. Once confirmed as a global state transaction, this _transaction fee_ is paid to the network to help support the [economic design](#economic-design) of the Solana blockchain.
```

Comment:
`transaction (aka collection of instructions)`
This is not technically correct, as a transaction has more than just instructions.

Reply:
While yes, their is more information like hte assorted meta data and what not, to simplify it on this page, I wrote it like that. 

Perhaps I could be more clear saying something like: `As each transaction (which contains one or more instructions)`

This way it is still clear/simple and more technically correct